core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -600,8 +589,6 @@ class TransactionStateManager(brokerId: Int,
                 recordsPerPartition,
                 updateCacheCallback,
                 delayedProduceLock = Some(newMetadata))
-
-              trace(s"Appended new metadata $newMetadata for transaction id $transactionalId with coordinator epoch $coordinatorEpoch to the local transaction log")
```

Comment:
Remove this as we will always printed it with `ack = -1`, hence this is meaning-less, and may be even misleading as it is not appended to local log even.